scripts/badger/arb_bvecvxcvx_pool.py

```diff
@@ -0,0 +1,63 @@
+from brownie import interface
+from sympy import Symbol
+from sympy.solvers import solve
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+
+TARGET = .55
+THRESHOLD = 5000
+
+
+def main():
+    """
+    arbitrage the bvecvxcvx factory pool. if bvecvx dominance is < TARGET, calc
+    how much bvecvx exactly should be added to reach TARGET. make sure that
+    the to be added bvecvx > THRESHOLD.
+    """
+
+    safe = GreatApeSafe(registry.eth.badger_wallets.treasury_ops_multisig)
+    safe.init_curve()
+
+    cvx = interface.ERC20(registry.eth.treasury_tokens.CVX, owner=safe.account)
+    bvecvx = interface.ISettV4h(
+        registry.eth.treasury_tokens.bveCVX, owner=safe.account
+    )
+    bvecvxcvx = interface.IStableSwap2Pool(
+        registry.eth.treasury_tokens['bveCVX-CVX-f'], owner=safe.account
+    )
+
+    safe.take_snapshot(tokens=[bvecvx.address, bvecvxcvx.address])
+
+    bal_cvx = cvx.balanceOf(bvecvxcvx)
+    bal_bvecvx = bvecvx.balanceOf(bvecvxcvx)
+    bal_total = bal_cvx + bal_bvecvx
```

Comment:
Here we're assuming that bveCVX will never increase in PPFS.  We can note this in a comment as it likely will not during the expected use of this script, or we can take into account PPFS, but should be apparent if used for reference down the line.